### PR TITLE
Adds Assently E-Sign custom connector for certification

### PR DESCRIPTION
**Pull request by kian-beetroot**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

**Comment by mamurshe** [comment]
Should this be in the certified connector folder instead?

**Comment by ptalrejagithub** [comment]
@mamurshe, yes this is certified connector.

File: custom-connectors/Assently E-Sign/Intro.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿
```
Comment (sriyen-msft) [inline custom-connectors/Assently E-Sign/Intro.md:1]: @kian-beetroot Please delete intro.md as it is not required for open sourcing your connector.

File: custom-connectors/Assently E-Sign/Readme.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿
```
Comment (sriyen-msft) [inline custom-connectors/Assently E-Sign/Readme.md:1]: @kian-beetroot You can refer below readme.md for reference.

[A Sample readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: custom-connectors/Assently E-Sign/Intro.md (hunk 1 above)
Comment (kian-beetroot) [reply custom-connectors/Assently E-Sign/Intro.md]: deleted as requested

File: custom-connectors/Assently E-Sign/Readme.md (hunk 2 above)
Comment (kian-beetroot) [reply custom-connectors/Assently E-Sign/Readme.md:1]: do we need to do any changes on the readme file?

**Review by mamurshe — CHANGES_REQUESTED** [review]
Please move this to certified-connectors folder

**Comment by kian-beetroot** [comment]
> Please move this to certified-connectors folder

done.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**